admissions/views.py

```diff
@@ -91,3 +91,17 @@ def get_programs(applications):
         program_set.add(application.program)
     program_set.add(ALL)
     return list(sorted(program_set))
+
+
+def reject(request, application_id):
+    application = HighSchoolApplication.objects.get(id=application_id)
+    application.application_status = "accepted"
```

Comment:
you can pass the status as a parameter to create only one view

Reply:
its not view, it is function, so i dont want to pass it, and also I already pass id as parameter

Reply:
> you can pass the status as a parameter to create only one view
its not view, it is function, so i dont want to pass it, and also I already pass id as parameter

Reply:
Its better if you take the status as parameter because this is duplicating the code.

Reply:
> Its better if you take the status as parameter because this is duplicating the code.

I think for now we keep my way. Since I already change the status as 0/1/2, then I dont want it messed with id as parameter in url